app/views/pages/text-filtering.html.erb

```diff
@@ -0,0 +1,65 @@
+<% provide(:title, 'Filtering the Full-Text Derivative File') %>
+
+<% cache do %>
+  <div class="container">
+    <h1 class="about_h1">Filtering the Full-Text Derivative File</h1>
+    <p class="about_p_italic">Tutorial by Ian Milligan (Archives Unleashed Team)</p>
+
+    <h3 class="about_h3">Introduction</h3>
+    <p class="about_p">You may want to filter the derivative files further, so that they for example:</p>
+    <ul class="about_ul">
+      <li class="about_li">Only contain websites from a certain date (i.e. all sites collected on July 16th, 2009)</li>
+      <li class="about_li">Only contain websites from a certain domain (i.e. the plain text of all "liberal.ca" websites)</li>
+      <li class="about_li">Only contain websites that have a certain keyword (i.e. every page that contains the word "elephant")</li>
```

Comment:
this list has a slightly small font then rest of the body (lines 11-13). It's a minor CSS change - noticed that this happens a few other times in various AUK pages - will open new issue to address this change

Reply:
OK, thanks @SamFritz!